scenarios/scenario3_5/infrastructure.tf

```diff
@@ -3,7 +3,7 @@ locals {
   workspace_prefix = terraform.workspace == "default" ? "" : "${terraform.workspace}-"
   key_pair = {
     public_key = var.public_key
-    key_name   = "${local.workspace_prefix}kp_${var.scenario}"
+    key_name   = "${local.workspace_prefix}kl_${var.scenario}"
```

Comment:
is it required?

Reply:
should be "${local.workspace_prefix}kp_${var.scenario}